src/NetTopologySuite/Geometries/ILineal.cs

```diff
@@ -11,4 +11,45 @@ public interface ILineal
     {
         
     }
-}
\ No newline at end of file
+
+    /// <summary>
+    /// Marker interface to identify all <c>Geometry</c> subclasses that have a <c>Dimension</c> of
+    /// <see cref="Dimension.Curve"/> and consist of <b>only</b> <c>1</c> component.
+    /// </summary>
+    public interface ICurve : ILineal
+    {
+        /// <summary>
+        /// Gets a value indicating that this <c>Curve</c> forms a ring.<br/>
+        /// A curve forms a ring if it <see cref="IsClosed"/> and <see cref="Geometry.IsSimple"/>
+        /// </summary>
+        bool IsRing { get; }
+
+        /// <summary>
+        /// Gets a value indicating that this <c>Curve</c> is closed.<br/>
+        /// A curve is closed if <c><see cref="StartPoint"/> == <see cref="EndPoint"/></c>.
+        /// </summary>
+        bool IsClosed { get; }
```

Comment:
These members essentially just wrap things that you can do with `CoordinateSequence` and probably don't belong on the interface proper.  They just make it more expensive to implement.

I guess `IsRing` isn't something that you can **directly** calculate with just the other members of the interface, but since it's required to be a `Geometry` subclass, it's still just:
```csharp
ICurve myCurve = GetCurveFromSomewhere();
bool isRing = myCurve.IsClosed && ((Geometry)myCurve).IsSimple;
```

Reply:
But it is defined as a property for `Curve`s in SFA.

Reply:
> But it is defined as a property for `Curve`s in SFA.

Yes, which is part of why I proposed making the base classes in a different comment.  If we're going to be introducing this part of SFA, then let's commit to doing so.  Using interfaces to mimic the relationship doesn't seem like a winning strategy...